app/dashboard/templates/dashboard.html

```diff
@@ -30,20 +32,24 @@
       {% include 'shared/nav_internal.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col-2" style="background: #eee; ">
+      <div class="col-12 col-md-2" style="background: #F2F6F9;">
         {% include 'shared/current_balance.html' %}
         {% include 'shared/sidebar_search.html' %}
       </div>
 
-      <div class="col body">
+      <div class="col-12 col-md-10 body">
         <div class="container-fluid" id="bounties">
           {% include 'shared/search_bar.html' %}
           <div class="row mt-2 mb-2">
-            <div class="col-10 ml-3 font-bigger-2" style="color: #3e24fb;padding-left: 1rem;">
-              <span id="modifiers">Funded Issues</span>
-              <span id=stats class="font-bigger-1 pl-1"></span>
+            <div class="col-12 offset-md-1 col-md-10 title-row">
+              <div class="col-4">
+                <span id="filter" class="font-title"></span>
+              </div>
+              <div class="col-8 bounty-info">
+                <span id="modifiers">Funded Issues</span>
+                <span id=stats class="font-body"></span>
```

Comment:
@thelostone-mc -- Shouldn't `id` here be wrapped in double quotes?

Reply:
Oh shoot :P thanks